CHANGELOG.asciidoc

```diff
@@ -64,15 +70,11 @@ https://github.com/elastic/beats/compare/1.0.0...master[Check the HEAD diff]
 ////////////////////////////////////////////////////////////
 
 [[release-notes-1.0.1]]
-=== Beats version 1.0.1 https://github.com/elastic/beats/compare/1.0.0...1.0.1[Check 1.0.1 diff]
+=== Beats version 1.0.1
+https://github.com/elastic/beats/compare/1.0.0...1.0.1[Check 1.0.0 diff]
```

Comment:
isn't there an issue with '...' and asciidoc converting these into one unicode symbol?